pipelines/test/ocp/aws/pipeline.yaml

```diff
@@ -0,0 +1,228 @@
+      name: additional-env
+      type: string
+    # Report Portal related parameters
+    - default: full-cycle-ocp-aws
+      description: Prefix of the launch name saved in report portal (nightly, username, manual, etc.). In case of release candidate testing use kuadrant-v<version>, rhcl-v<version>, or authorino-v<version>
+      name: launch-name
+      type: string
+    - default: ""
+      description: Optional launch description for Report Portal
+      name: launch-description
+      type: string
+    - default: testsuite
+      description: Report Portal Project to store test results (testsuite or nightly-testsuite typically)
+      name: rp-project
+      type: string
+    - default: true
+      description: If set to 'true' upload test results to Report Portal
+      name: upload-results
+      type: string
+    # Cleanup
+    - default: false
+      description: If set to 'true' cleanup will be done
+      name: cleanup
+      type: string
+  workspaces:
+    - name: shared-workspace
+      description: "Workspace to store cluster state files"
+  tasks:
+    - name: provision-ocp-aws
+      params:
+        - name: cluster-name
+          value: $(params.cluster-name)
+        - name: region
+          value: $(params.region)
+        - name: master-size
+          value: $(params.master-size)
+        - name: worker-size
+          value: $(params.worker-size)
+        - name: installer-arch
+          value: $(params.installer-arch)
+      workspaces:
+        - name: shared-workspace
+          workspace: shared-workspace
+      taskRef:
+        kind: Task
+        name: provision-ocp-aws
+    - name: helm-deploy
+      retries: 3
+      params:
+        - name: index-image
+          value: $(params.index-image)
+        - name: channel
+          value: $(params.channel)
+        - name: istio-provider
+          value: $(params.istio-provider)
+        - name: operator-name
+          value: $(params.operator-name)
+        - name: kubeconfig-path
+          value: $(tasks.provision-ocp-aws.results.kubeconfig-path)
+        - name: gateway-crd
+          value: $(params.gateway-crd)
+        - name: keycloak-channel
+          value: $(params.keycloak-channel)
+        - name: additional-helm-flags
+          value: $(params.additional-helm-flags)
+        - name: additional-helm-tools-flags
+          value: $(params.additional-helm-tools-flags)
+      runAfter:
+        - provision-ocp-aws
+      taskRef:
+        kind: Task
+        name: helm-deploy
+      workspaces:
+        - name: shared-workspace
+          workspace: shared-workspace
+    - name: do-custom-updates
+      params:
+        - name: kubeconfig-path
+          value: $(tasks.kubectl-login.results.kubeconfig-path)
+      runAfter:
+        - helm-deploy
+      taskRef:
+        kind: Task
+        name: do-custom-updates
```

Comment:
Missing workspace definition for `do-custom-updates` Task. See
https://github.com/Kuadrant/testsuite-pipelines/blob/main/pipelines/test/osd/pipeline.yaml#L404-L406